content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -283,7 +283,15 @@ systemctl restart containerd
 
 To use the `systemd` cgroup driver, set `plugins.cri.systemd_cgroup = true` in `/etc/containerd/config.toml`.
 When using kubeadm, manually configure the
-[cgroup driver for kubelet](/docs/setup/independent/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-master-node)
+<<<<<<< HEAD
```

Comment:
This looks like merge that got left in by accident.

Reply:
Thank you it was a miss. Fixed it now.